Entity/Banner.php

```diff
@@ -0,0 +1,425 @@
+<?php
+
+namespace WebtownPhp\BannerBundle\Entity;
+
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Banner
+ *
+ * @ORM\Table()
+ * @ORM\Entity(repositoryClass="Webtown\EmailBundle\Entity\BannerRepository")
+ */
+class Banner
+{
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="id", type="integer", nullable=false)
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="name", type="string", length=255, nullable=false)
+     */
+    private $name;
+
+    /**
```

Comment:
Készüljön rövid leírás a mezőkhöz, hogy melyikben mit tárolunk, ha nem teljesen egyértelmű. Pl: `The banner target URL`. És ez kerüljön bele az adatbázisba is (lásd `options`):

```php
    /**
     * The banner target URL.
     *
     * @var string $target
     *
     * @ORM\Column(name="target", type="string", length=255, nullable=false, options={
     *          "comment": "The banner target URL."})
     */
    protected $target;
```